test.txt

```diff
@@ -0,0 +1,2 @@
+Début
```

Comment:
What is the purpose of this file?

Reply:
I forgot to delete it while commiting my changes, it was for testing things at the beginning. I deleted it.